src/SmartComponents/ResourceOptimization/ResourceOptimizationCard.js

```diff
@@ -25,6 +25,9 @@ const ResourceOptimizationCard = ({
     }, [fetchRosIsConfigured]);
 
     const navigateTo = useChromePush();
+    const waitingForDataUrl = `${UI_BASE}/ros?state=waiting`;
+    const suggestionsUrl = `${UI_BASE}/ros?state=suggestions`;
```

Comment:
```suggestion
    const waitingForDataUrl = `${UI_BASE}/ros?with_waiting_for_data=true`;
    const suggestionsUrl = `${UI_BASE}/ros?with_suggestions=true`;
```

Instead of `state` key in query param, will it be good idea to use query string flags here as those are not state values?
Example - `${UI_BASE}/ros?with_suggestions=true`;

Any thoughts?

Reply:
I agree @kgaikwad it will make it more readable, I will update it and push the changes soon.